e2e-tests/tests/infra/spawnStudio.ts

```diff
@@ -23,7 +23,15 @@ export default async function spawnStudio(
   const port = await getPort(testInfo);
   const child = spawn(
     "npx",
-    ["studio", "--port", port.toString(), "--root", rootDir],
+    [
+      "studio",
+      "--port",
+      port.toString(),
+      "--root",
+      rootDir,
+      "--",
+      "--noGenerateTestData",
```

Comment:
could we remove these since these args don't exist anymore

Reply:
Yep